knife-solo.gemspec

```diff
@@ -31,6 +30,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'coveralls'
 
   s.add_dependency 'chef',     '>= 10.20'
-  s.add_dependency 'net-ssh',  '~> 2.7', '< 3.0'
-  s.add_dependency 'erubis',   '~> 2.7.0'
+  s.add_dependency 'net-ssh',  '~> 2.6'
```

Comment:
I suspect at the time it would have been to match other dependencies. But it's been awhile so probably wouldn't hurt to try `">= 2.9", "< 4.0"` and see if anything breaks. If you set up the integration test suite (uses EC2 instances) the coverage is pretty comprehensive.